pipeline/metadata/6.0.yaml

```diff
@@ -1091,3 +1091,19 @@ suites:
       - ibmc
       - dmfg
       - stage-5
+
+  - name: "test-rgw-ssl-bucket-notifications"
+    suite: "suites/quincy/rgw/tier-2_rgw_ssl-test-bucket-notifications.yaml"
+    global-conf: "conf/pacific/rgw/tier-0_rgw.yaml"
```

Comment:
Can we use conf file from quicy: https://github.com/red-hat-storage/cephci/blob/master/conf/quincy/rgw/tier-0_rgw.yaml

Reply:
@ckulal ,updated to use quincy conf file